docs/src/theme/prism-include-languages.js

```diff
@@ -4,27 +4,25 @@ export default function prismIncludeLanguages(PrismObject) {
     themeConfig: {prism},
   } = siteConfig;
   const {additionalLanguages} = prism;
-  // Prism components work on the Prism instance on the window, while prism-
-  // react-renderer uses its own Prism instance. We temporarily mount the
-  // instance onto window, import components to enhance it, then remove it to
-  // avoid polluting global namespace.
-  // You can mutate PrismObject: registering plugins, deleting languages... As
-  // long as you don't re-assign it
+  
+  const PrismBefore = globalThis.Prism;
   globalThis.Prism = PrismObject;
   additionalLanguages.forEach((lang) => {
     // eslint-disable-next-line global-require, import/no-dynamic-require
     require(`prismjs/components/prism-${lang}`);
   });
-  const rustLanguage = Prism.languages.rust;
   Prism.languages["rust,ignore"] = Prism.languages.rust;
 
   const origTokenize = PrismObject.tokenize;
-  PrismObject.tokenize = (text, grammar) => {
-    if (grammar == rustLanguage) {
-      text = text.split("\n").filter(line => !line.startsWith("# ")).join("\n");
+  PrismObject.hooks.add("after-tokenize", function(env) {
+    if (env.language == "rust") {
```

Comment:
Should this be `=== Prism.languages.rust`? (should at least be triple equal for style)

Reply:
Not in this case, it's just a string identifier rather than an object.